packages/vm/src/identity_digest.rs

```diff
@@ -0,0 +1,43 @@
+//! Dummy 256-bits Digest impl.
+//! This digest stores/accepts a value of the proper length.
+//! To be used for / with already hashed values, just to comply with the Digest contract.
+//!
+//! Adapted from `sha2` [sha256.rs](https://github.com/RustCrypto/hashes/blob/master/sha2/src/sha256.rs)
+use sha2::digest::consts::U32;
+use sha2::digest::generic_array::GenericArray;
+use sha2::digest::{FixedOutputDirty, Output, Reset, Update};
```

Comment:
We can take all of those from the `digest` crate. No need for a sha2 dependency in non-test code

Reply:
Fixed. This was an introduced during development.